python/ml4ir/applications/ranking/tests/test_metrics.py

```diff
@@ -94,3 +97,15 @@ def test_model_training(self):
         for gold_metric_name, gold_metric_val in GOLD_METRICS.items():
             assert gold_metric_name in metrics
             assert np.isclose(metrics[gold_metric_name], gold_metric_val, rtol=0.01)
+
+
+def test_top_5_categorical_accuracy():
+    """Test for calculating top_5_categorical_accuracy metric"""
+    feature_config = os.path.join(ROOT_DATA_DIR, "configs", "feature_config.yaml")
+    feature_config = LocalIO().read_yaml(feature_config)
+    metric = Top5CategoricalAccuracy(feature_config=feature_config, metadata_features={})
```

Comment:
I recommend passing in a None object for feature_config here instead of loading one. @balikasg